lib/experimental/fonts/font-library/class-wp-font-library.php

```diff
@@ -27,6 +27,64 @@ class WP_Font_Library {
 		'woff2' => 'font/woff2',
 	);
 
+	/**
+	 * Font collections.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @var array
+	 */
+	private static $collections = array();
+
+	/**
+	 * Register a new font collection.
+	 *
+	 * @since 6.4.0
+	 *
+	 * @param string $id Font collection id.
+	 * @param array  $config Font collection config options.
+	 * @return WP_Font_Collection|WP_Error A font collection is it was registered succesfully and a WP_Error otherwise.
+	 */
+	public static function register_font_collection( $id, $config ) {
+		if ( array_key_exists( $id, self::$collections ) ) {
```

Comment:
```suggestion
		if ( isset( self::$collections[ $id ] ) ) {
```

Tiny tiny tiny micro-optimization.